#forest_path: '/home/stfn/dev/hf_mc/evaluations/scale_invariance_resized/train_rl/_finalfeature_gpu/1/forest/treetable_'
#forest_path: '/home/stfn/recognition_database/forests/huiii/'
num_trees: 15
patch_widht: 10
patch_height: 10
kernel_smooth: 15
kernel_vote: 20
kernel_sigma: 2
threshold_hierarchy: 1.5
max_candidates: 20
detection_threshold: 0.1
do_backprojection: true
cand_max_width: 320
cand_max_height: 240